assets/js/app.js

```diff
@@ -23,3 +23,4 @@ import "phoenix_html"
 
 import "./header"
 import "./filter_players"
+import "./filter_trade_form"
```

Comment:
Missing semicolon.

[Read more about it here.](http://eslint.org/docs/rules/semi)